fixtures/autofix-unsupported_v1.yml

```diff
@@ -0,0 +1,72 @@
+# This is a test yaml file
+# to be autofixed
+
+%YAML   1.1
+%TAG    !   !foo
+%TAG    !yaml!  tag:yaml.org,2002:
+
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: test-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /
+  creationTimestamp: null
+spec:
+  rules: #This is a comment 1
+  - http:
+      paths:
+      - backend:
+          serviceName: test
+          servicePort: 80
+        path: /testpath
+status:
+  loadBalancer: {}
+#This is a comment 2
+---
+#This is a comment 3
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: test-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /
+  creationTimestamp: null
+spec:
+  rules:
+  - http:
+      paths:
+      - backend: #This is a comment 4
```

Comment:
When running autofix, this comment gets swallowed.

Reply:
https://github.com/Shopify/kubeaudit/blob/autofix/cmd/autofix.util.go#L415 http was missing in ports case, has been added. Works like a charm now